tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -172,6 +172,8 @@ shader, this value can be used as desired.
 +----------------------------------------+--------------------------------------------------------+
 | in vec3 **CAMERA_DIRECTION_WORLD**     | Camera world space direction.                          |
 +----------------------------------------+--------------------------------------------------------+
+| in int **CAMERA_VISIBLE_LAYERS**        | Visible layers indicator for the camera.              |
```

Comment:
```suggestion
| in int **CAMERA_VISIBLE_LAYERS**       | Visible layers indicator for the camera.               |
```

Reply:
I would reword the description as well to say "Cull layers of the camera rendering the current pass." instead.